mod/profiles.php

```diff
@@ -747,15 +743,11 @@ function profiles_content(App $a) {
 		call_hooks('profile_edit', $arr);
 
 		return $o;
-	}
+	} else {
 
-	//Profiles list.
-	else {
-
-		//If we don't support multi profiles, don't display this list.
-		if(!feature_enabled(local_user(),'multi_profiles')){
-			$r = q(
-				"SELECT * FROM `profile` WHERE `uid` = %d AND `is-default`=1",
+		// If we don't support multi profiles, don't display this list.
+		if (!feature_enabled(local_user(),'multi_profiles')){
```

Comment:
Standards: Can you please add a space after commas and before the opening brace?